### PR TITLE
Do not rely on changing Array.prototype

### PR DESCRIPTION
**Pull request by jdormit**
I removed the `Array.prototype.last` function, instead defining inline variables to represent the last array element when necessary.

### CODE REVIEW

**Comment by jdormit** [comment]
To clarify - adding `Array.prototype.last` breaks `for (var x in array)`, because that for loop will always end with `x = last ()`.

File: index.js [hunk 1]
```diff
@@ -105,10 +101,10 @@
...
-              span.responses.last()[category]++;
-              span.responses.last().count++;
-              span.responses.last().mean = span.responses.last().mean + ((responseTime - span.responses.last().mean) / span.responses.last().count);
+              last.timestamp / 1000 + span.interval > Date.now() / 1000) {
+              last[category]++;
+              last.count++;
+              last.mean = last.mean + ((responseTime - last.mean) / last.count);
             } else {
               span.responses.push({
                 '2': category === 2 ? 1 : 0,
```
Comment (RafalWilinski) [inline index.js]: In theory everything is okay but have you tested it in action?

**Comment by RafalWilinski** [comment]
Before merging I'll test that.

**Comment by jdormit** [comment]
Yup, it worked fine when I tested it - but double checking never hurts!

**Comment by jdormit** [comment]
In case anyone is browsing the issues, this references #19

**Pull request merged**